source/less/blocks/videopresentation.less

```
@@ -6,8 +6,21 @@
 
 .videopresentation {
   margin: 62px 0 120px 0;
+  position: relative;
 }
 
 .videopresentation__video {
   padding-top: 64px;
+  width: 100%;
+  object-fit: contain;
+  @media (min-width: @tablet-width) {
+    min-height: 500px;
```

```suggestion
    height: 500px;
```
почему не просто heigth?